pyproject.toml

```diff
@@ -54,7 +54,9 @@ pygsheets = "^2.0.4"
 PyYAML = "^6.0.0"
 rdflib = "^6.0.0"
 setuptools = "^66.0.0"
-synapseclient = "4.4.1"
+#synapseclient = "4.4.1"
+# GIT dependency for synapseclient
+synapseclient = { git = "https://github.com/Sage-Bionetworks/synapsePythonClient.git", branch = "fds-2386-allow-global-client-cache-disable" }
```

Comment:
I have a few fixes I've found and added for SYNPY that will need to be included. For now I am pointing to the GIT branch that has the changes, but will plan to release this before I merged this into develop.